README.md

```diff
@@ -37,8 +38,19 @@ level of precision is better left to other tools.
   **Deprecation: an offset may be appended, e.g. `+0700` or `-0400`, but you
   should use `location` instead.**
 
-* `days`: *Optional.* Run only on these day(s). Supported days are: `Sunday`,
-  `Monday`, `Tuesday`, `Wednesday`, `Thursday`, `Friday` and `Saturday`.
+  To explicitly represent a full calendar day, set `start` and `stop` to
+  the same value.
+
+  e.g.
+
+  ```
+  start: 6:00 AM
+  stop: 6:00 AM
+  ```
```

Comment:
👍 This is the detail I was missing that made everything make sense. It's a little strange to specify the same value for both, since the value itself doesn't matter, but it at least intuitively makes sense with the 12AM default.

Just for the sake of argument, it seems like it could be technically valid to specify `start` without `stop` and vice versa.

This would mean "every 10 minutes from 7PM to the end of the day":

```yaml
interval: 10m
start: 7PM
```

And this would mean "every 10 minutes from the start of the day until 9AM":

```yaml
interval: 10m
stop: 9AM
```

Is that right?

Not arguing for allowing it necessarily, just checking my logic.

In any case it might be worth mentioning that when not specified they both default to the same value, 12AM, and that that's how the resource achieves the "1 version per day" behavior when nothing is specified.

Reply:
> Just for the sake of argument, it seems like it could be technically valid to specify start without stop and vice versa.

Yeah, that's why I got a bit confused by your comment about tossing the validation logic. You _**could**_ throw out all the validation logic if you really want to.

> This would mean "every 10 minutes from 7PM to the end of the day":
> And this would mean "every 10 minutes from the start of the day until 9AM":
> Is that right?

👍  Nailed it

Just pushed another README update to try to clarify the `start`/`stop` defaults. Hopefully that helps (and doesn't confuse things even more).